### PR TITLE
Maddie- object endpoints profile values

### PR DESCRIPTION
**Pull request by maddiechase**
added endpoint info for profile values for
Companies
Contacts
Affiliations
Prospects
Issues
Milestones
Staff
Jobs
Invoice
Contracts

### CODE REVIEW

File: source/includes/endpoints/_affiliations.md [hunk 1]
```diff
@@ -423,7 +472,21 @@ This request will delete the affiliation identified by its `affiliation_id`. Thi
...
-This request returns a list of [profile field values](#the-profile-value-object) of an [affiliation](#the-affiliation-object), specified by its `affiliation_id`. This is the request [`GET /{object}/{object_id}/profiles/values`](#retrieve-a-list-of-profile-values), where the object is "affiliations" whose id is `{affiliation_id}`.
+This request returns a list of [profile field values](#the-profile-value-object) of an [affiliation](#the-affiliation-object), 
+specified by its `affiliation_id`. This is the request 
+[`GET/{object}/{object_id}/profiles/values`](#retrieve-a-list-of-profile-values), where the object is "affiliations" whose id
+is `{affiliation_id}`.
+
+
+
+
+### List all Profile Field Values on an Affiliation
```
Comment (maddiechase) [inline source/includes/endpoints/_affiliations.md]: @matthewhughes112  Should this title be changed?
`List Affiliation Profile Field Values`

File: source/includes/endpoints/_contracts.md [hunk 2]
```diff
@@ -682,7 +780,49 @@ This request returns a list of [collections](#resources) against a [company](#th
...
+`GET /contracts/profiles/values`
+
+This request returns a list of all [profile field values](#the-profile-value-object) of a [contract](#the-contract-object). 
+This is the request [`GET /{object}/profiles/values`](#list-profile-values), where the object is "contracts".
+
+
+
+
+
+### List Affiliation Profile Fields
```
Comment (matthewhughes112) [inline source/includes/endpoints/_contracts.md]: Typo in `Affiliation` in the title here

File: source/includes/endpoints/_contracts.md [hunk 3]
```diff
@@ -689,6 +689,42 @@ This request uploads a [resource](#resources) to a collection, specified by its
...
+> See the [profiles section](#retrieve-a-list-of-profile-fields) for a sample request
+
+`GET /contracts/profiles/fields`
+
+This request returns a list of all [profile field values](#the-profile-value-object) of a [contract](#the-contract-object). This is the request [`GET /{object}/profiles/values`](#list-profile-values), where the object is "contracts".
```
Comment (matthewhughes112) [inline source/includes/endpoints/_contracts.md]: Description appears to be for `GET /contracts/profiles/values` endpoint.

File: source/includes/endpoints/_contracts.md [hunk 4]
```diff
@@ -682,7 +780,49 @@ This request returns a list of [collections](#resources) against a [company](#th
...
+
+
+
+
+### List a Contract's Profile Field Values
+> See the [profiles section](#retrieve-a-list-of-profile-values) for a sample request
+
+`GET /contract/{contract_id}/profiles/values`
+
+This request returns a list of [profile field values](#the-profile-value-object) on [contracts](#the-contract-object),
```
Comment (matthewhughes112) [inline source/includes/endpoints/_contracts.md]: should this be `of a contract`

File: source/includes/endpoints/_contracts.md [hunk 5]
```diff
@@ -682,7 +780,49 @@ This request returns a list of [collections](#resources) against a [company](#th
...
+
+
+
+
+### List all Profile Field Values on a Contract
+> See the [profiles section](#list-profile-values) for a sample request
+
+`GET /contracts/profiles/values`
+
+This request returns a list of all [profile field values](#the-profile-value-object) of a [contract](#the-contract-object).
```
Comment (matthewhughes112) [inline source/includes/endpoints/_contracts.md]: Should this be `on contracts`?

**Pull request merged**